atd-vzd/schema/afd__incidents.sql

```diff
@@ -0,0 +1,65 @@
+DROP TABLE public.afd__incidents;
+
+-- Table Definition
+CREATE TABLE "public"."afd__incidents" (
+    id serial primary key,
+    incident_number integer,
+    crash_id integer default null,
+    unparsed_ems_incident_number text default null,
+    ems_incident_numbers integer[] default null,
```

Comment:
We can get 0, 1, or more incident numbers here, so we're storing them in an array. Also keeping the unparsed value, just in case.